### DISCUSSION TITLE
cognate classes

### DISCUSSION THREAD

**Discussion by gerhardJaeger** [lingpy/are-sounds-sound-paper#1] [Announcements]
Do we have cognate class data for these language sets as well?

**Comment by LinguList** [comment]
Yes, cognate sets are a requirement to get the correspondence patterns. You find them already as "cogid" in the folder "datasets". These are all done by experts (I deliberately choose the ten datasets for this reason, or better: they were chosen in a study that was now accepted for a workshop at the EACL, where we use the trimming algorithm to cut down those columns in the alignments that are most likely reflecting morphological differences).
